src/test/kotlin/sh/hsp/techtree/graphviz/DSLConverterTest.kt

```diff
@@ -39,11 +36,24 @@ class DSLConverterTest {
             )
         )
 
-        val converter = DSLConverter()
-
-        val result = converter.convert(model)
+        val result = sut.convert(model)
 
         "digraph { graph [splines=\"ortho\"] node [shape=\"box\"] edge [dir=\"back\"] nodeA nodeA -> nodeB nodeA -> nodeC nodeB nodeC }".shouldBe(result)
     }
 
+    @Test
+    fun givenSpacesInTitleShouldRegisterNodesInCorrectGraphvizDSL() {
+        val model = TreeModel(
+            listOf(
+                TreeNode("Big dog", listOf("Small dog")),
+                TreeNode("Small dog"),
+                TreeNode("Big cat")
+            )
+        )
+
+        val result = sut.convert(model)
+
+        "digraph { graph [splines=\"ortho\"] node [shape=\"box\"] edge [dir=\"back\"] \"Big dog\" \"Big dog\" -> \"Small dog\" \"Small dog\" \"Big cat\" }".shouldBe(result)
```

Comment:
To powinien być plik w resources, który jest wczytyway do testu.
Jak Cię interesuje to możesz wykombinować jak zrobić taska w gradlu, który go wygeneruje z TreeModelu.

Ten test po prostu powinien failować, jeżeli ktoś coś zmieni, a nie pomyśli, że zmienia się wynikowy output. Normalnie powinno go prosto dać się przegenerować.